### ISSUE TITLE
Build fails with error about opentk_1_0.GameViewBase is defined multiple times

### ISSUE THREAD

**Issue by pfafft33** [xamarin/xamarin-android#5330]
### Steps to Reproduce

1. Updated to VS Community 2019 v 16.8.1
2. Tried to build my solution which was building and running just fine in v 16.7.7
3. Error:  C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Xamarin\Android\Xamarin.Android.D8.targets(78,5): error MSB6006: "java.exe" exited with code 1.

### Expected Behavior
Expect a successful build

### Actual Behavior

```
5>  C:\Program Files\Android\Jdk\microsoft_dist_openjdk_1.8.0.25\bin\java.exe -Xmx1G -classpath "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Xamarin\Android\r8.jar" com.android.tools.r8.D8 --debug --min-api 19 --output obj\Debug\100\android\bin\ --lib "C:\Program Files (x86)\Android\android-sdk\platforms\android-29\android.jar" obj\Debug\100\android\bin\classes.zip "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\MonoAndroid\v10.0\mono.android.jar" "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Xamarin\Android\java_runtime_fastdev.jar" C:\Users\Terence\.nuget\packages\xamarin.androidx.annotation\1.1.0.6\buildTransitive\monoandroid90\..\..\jar\androidx.annotation.annotation.jar C:\Users\Terence\.nuget\packages\xamarin.androidx.collection\1.1.0.5\buildTransitive\monoandroid90\..\..\jar\androidx.collection.collection.jar C:\Users\Terence\.nuget\packages\xamarin.androidx.lifecycle.common\2.2.0.3\buildTransitive\monoandroid90\..\..\jar\androidx.lifecycle.lifecycle-common.jar C:\Users\Terence\.nuget\packages\xamarin.androidx.arch.core.common\2.1.0.6\buildTransitive\monoandroid90\..\..\jar\androidx.arch.core.core-common.jar obj\Debug\100\lp\0\jl\formsviewgroup.jar obj\Debug\100\lp\52\jl\classes.jar obj\Debug\100\lp\53\jl\classes.jar obj\Debug\100\lp\54\jl\classes.jar obj\Debug\100\lp\55\jl\classes.jar obj\Debug\100\lp\56\jl\classes.jar obj\Debug\100\lp\57\jl\classes.jar obj\Debug\100\lp\58\jl\classes.jar obj\Debug\100\lp\59\jl\classes.jar obj\Debug\100\lp\60\jl\classes.jar obj\Debug\100\lp\61\jl\classes.jar obj\Debug\100\lp\62\jl\classes.jar obj\Debug\100\lp\63\jl\classes.jar obj\Debug\100\lp\64\jl\classes.jar obj\Debug\100\lp\65\jl\classes.jar obj\Debug\100\lp\66\jl\classes.jar obj\Debug\100\lp\67\jl\classes.jar obj\Debug\100\lp\68\jl\classes.jar obj\Debug\100\lp\69\jl\classes.jar obj\Debug\100\lp\70\jl\classes.jar obj\Debug\100\lp\71\jl\classes.jar obj\Debug\100\lp\72\jl\classes.jar obj\Debug\100\lp\73\jl\classes.jar obj\Debug\100\lp\74\jl\classes.jar obj\Debug\100\lp\75\jl\classes.jar obj\Debug\100\lp\76\jl\classes.jar obj\Debug\100\lp\77\jl\classes.jar obj\Debug\100\lp\78\jl\classes.jar obj\Debug\100\lp\79\jl\classes.jar obj\Debug\100\lp\80\jl\classes.jar obj\Debug\100\lp\81\jl\classes.jar obj\Debug\100\lp\82\jl\classes.jar obj\Debug\100\lp\83\jl\classes.jar obj\Debug\100\lp\84\jl\classes.jar obj\Debug\100\lp\85\jl\classes.jar obj\Debug\100\lp\86\jl\classes.jar obj\Debug\100\lp\88\jl\classes.jar obj\Debug\100\lp\89\jl\classes.jar obj\Debug\100\lp\90\jl\classes.jar  (TaskId:382)
5>  Error in obj\Debug\100\android\bin\classes.zip:opentk_1_0/GameViewBase.class: (TaskId:382)
5>  Type opentk_1_0.GameViewBase is defined multiple times: obj\Debug\100\android\bin\classes.zip:opentk_1_0/GameViewBase.class, C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\MonoAndroid\v10.0\mono.android.jar:opentk_1_0/GameViewBase.class (TaskId:382)
5>  Compilation failed (TaskId:382)
```
Got error:  C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Xamarin\Android\Xamarin.Android.D8.targets(78,5): error MSB6006: "java.exe" exited with code 1.

### Version Information

Microsoft Visual Studio Community 2019
Version 16.8.2
VisualStudio.16.Release/16.8.2+30717.126
Microsoft .NET Framework
Version 4.8.04084

<details>
Installed Version: Community

Visual C++ 2019   00435-60000-00000-AA349
Microsoft Visual C++ 2019

.NET Portability Analyzer   1.1.10808.0
Evaluates portability of assemblies across .NET platforms.

ASP.NET and Web Tools 2019   16.8.553.28003
ASP.NET and Web Tools 2019

Azure App Service Tools v3.0.0   16.8.553.28003
Azure App Service Tools v3.0.0

C# Tools   3.8.0-5.20567.16+53c5d7d3cf13d88978744a32a27c5f8350a8400a
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Common Azure Tools   1.10
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

Dotfuscator Community Edition   5.44.0.10087-6530a8d80a
PreEmptive Protection - Dotfuscator CE

Extensibility Message Bus   1.2.6 (master@34d6af2)
Provides common messaging-based MEF services for loosely coupled Visual Studio extension components communication and integration.

IntelliCode Extension   1.0
IntelliCode Visual Studio Extension Detailed Info

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

Microsoft Visual C++ Wizards   1.0
Microsoft Visual C++ Wizards

Microsoft Visual Studio VC Package   1.0
Microsoft Visual Studio VC Package

Mono Debugging for Visual Studio   16.8.43 (00471f8)
Support for debugging Mono processes with Visual Studio.

NuGet Package Manager   5.8.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit https://docs.nuget.org/

ProjectServicesPackage Extension   1.0
ProjectServicesPackage Visual Studio Extension Detailed Info

Test Adapter for Boost.Test   1.0
Enables Visual Studio's testing tools with unit tests written for Boost.Test.  The use terms and Third Party Notices are available in the extension installation directory.

Test Adapter for Google Test   1.0
Enables Visual Studio's testing tools with unit tests written for Google Test.  The use terms and Third Party Notices are available in the extension installation directory.

TypeScript Tools   16.0.21016.2001
TypeScript Tools for Microsoft Visual Studio

Visual Basic Tools   3.8.0-5.20567.16+53c5d7d3cf13d88978744a32a27c5f8350a8400a
Visual Basic components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Visual F# Tools   16.8.0-beta.20507.4+da6be68280c89131cdba2045525b80890401defd
Microsoft Visual F# Tools

Visual Studio Code Debug Adapter Host Package   1.0
Interop layer for hosting Visual Studio Code debug adapters in Visual Studio

Visual Studio Tools for CMake   1.0
Visual Studio Tools for CMake

VisualStudio.DeviceLog   1.0
Information about my package

VisualStudio.Foo   1.0
Information about my package

VisualStudio.Mac   1.0
Mac Extension for Visual Studio

Xamarin   16.8.000.255 (d16-8@d002176)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin Designer   16.8.0.507 (remotes/origin/d16-8@e87b24884)
Visual Studio extension to enable Xamarin Designer tools in Visual Studio.

Xamarin Templates   16.8.112 (86385a3)
Templates for building iOS, Android, and Windows apps with Xamarin and Xamarin.Forms.

Xamarin.Android SDK   11.1.0.17 (d16-8/c0e2b8e)
Xamarin.Android Reference Assemblies and MSBuild support.
    Mono: be2226b
    Java.Interop: xamarin/java.interop/d16-8@79d9533
    ProGuard: Guardsquare/proguard/proguard6.2.2@ebe9000
    SQLite: xamarin/sqlite/3.32.1@1a3276b
    Xamarin.Android Tools: xamarin/xamarin-android-tools/d16-8@2fb1cbc

Xamarin.iOS and Xamarin.Mac SDK   14.4.1.3 (e30c41de3)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.

</details>
### Log File

Build log file (in Diagnostic mode) attached below:
[Output-Build-20201124-1132.txt](https://github.com/xamarin/xamarin-android/files/5591750/Output-Build-20201124-1132.txt)

Android project's csproj file attached below:

[MedLemnMobile.Android.csproj.txt](https://github.com/xamarin/xamarin-android/files/5591766/MedLemnMobile.Android.csproj.txt)

Shared-code project's csproj file attached below:
[MedLemnMobile.csproj.txt](https://github.com/xamarin/xamarin-android/files/5591777/MedLemnMobile.csproj.txt)

**Comment by dellis1972** [comment]
You probably shouldn't reference OpenTK from the shared code project. It's not a netstandard 2.0 compatible assembly. 
You should only reference it from an Android Assembly. 

If you need to use OpenGL for the shared library perhaps you can try multi-targeting the shared code project to target both `netstandard2.0;monodroid'. 

is the shared code used on any other platforms? (iOS ? ).

**Comment by pfafft33** [comment]
I tried

<TargetFrameworks>netstandard2.0;monodroid</TargetFrameworks>

and got the error

C:\Program Files\dotnet\sdk\5.0.100\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.TargetFrameworkInference.targets(92,5): error NETSDK1013: The TargetFramework value 'monodroid' was not recognized. It may be misspelled. If not, then the TargetFrameworkIdentifier and/or TargetFrameworkVersion properties must be specified explicitly.

**Comment by dellis1972** [comment]
@pfafft33 try `monodroid90` .

**Comment by pfafft33** [comment]
I tried

netstandard2.0;monodroid90

and got the error

C:\Program Files\dotnet\sdk\5.0.100\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.TargetFrameworkInference.targets(92,5): error NETSDK1013: The TargetFramework value 'monodroid90' was not recognized. It may be misspelled. If not, then the TargetFrameworkIdentifier and/or TargetFrameworkVersion properties must be specified explicitly.

**Comment by pfafft33** [comment]
I solved it!!!
Looking at the Diagnostic Build Output, it was complaining about the type opentk_1_0.GameViewBase being defined multiple times. So, I got the idea that maybe the opentk assembly was getting copied around into multiple places.
This led me to discover that the "Copy Local" property of the assembly reference to OpenTK was not set - it was BLANK.

First of all, THIS IS A BUG!

The IDE should NOT allow a property with a yes/no value to ever be BLANK. This is confusing.
So, I changed the value of Copy Local to NO, and my app builds and runs just fine again, as it did BEFORE I upgraded to VS 16.,8.1

Thanks for your help!
